### PR TITLE
[MM-38468] - Webapp/System Console: Check if help text about customized Support Email is still accurate with the removal of tutorial screen

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR updates text on the support email description in the console

#### Ticket Link
[MM-38468](https://mattermost.atlassian.net/browse/MM-38468)

```release-note
NONE

```

### CODE REVIEW

**Review by marianunez — APPROVED** [review]

File: i18n/en_AU.json [hunk 1]
```diff
@@ -2105,7 +2105,7 @@
   "admin.sql.traceTitle": "SQL Statement Logging: ",
   "admin.support.aboutDesc": "The URL for the About link on the Mattermost login and sign-up pages. If this field is empty, the About link is hidden from users.",
   "admin.support.aboutTitle": "About Link:",
-  "admin.support.emailHelp": "Email address displayed on email notifications and during tutorial for end users to ask support questions.",
+  "admin.support.emailHelp": "Email address displayed on email notifications.",
```
Comment (saturninoabril) [inline i18n/en_AU.json:2108]: Please revert change to this file. We're only suppose to change from `en.json`. This (en_AU) and other locales will be handled separately by the Localization/Translation team.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @AGMETEOR! LGTM, except for one request.

**Review by pablovelezvidal — APPROVED** [review]
LGTM!

File: i18n/en_AU.json (hunk 1 above)
Comment (AGMETEOR) [reply i18n/en_AU.json]: I have reverted it

**Review by saturninoabril — APPROVED** [review]
Thanks @AGMETEOR. Tested and passed. LGTM.

**Pull request merged**